JASP-Engine/JASP/R/auditClassicalBenfordsLaw.R

```diff
@@ -304,9 +304,9 @@ auditClassicalBenfordsLaw <- function(jaspResults, dataset, options, ...){
                                 "uniform" = "the uniform distribution")
 
   message <- base::switch(options[["digits"]],
-                          "first" = gettextf("The null hypothesis specifies that the distribution of first numbers (1 - 9) in the population conforms to %1$s.", distribution),
-                          "firstSecond" = gettextf("The null hypothesis specifies that the distribution of the first two numbers (10 - 99) in the population conforms to %1$s.", distribution),
-                          "last" = gettextf("The null hypothesis specifies that the distribution of last numbers (1 - 9) in the population conforms to %1$s." ,distribution))
+                          "first" = gettextf("The null hypothesis specifies that the distribution of first digits (1 - 9) in the data set conforms to %1$s.", distribution),
+                          "firstSecond" = gettextf("The null hypothesis specifies that the distribution of the first two digits (10 - 99) in the data set conforms to %1$s.", distribution),
+                          "last" = gettextf("The null hypothesis specifies that the distribution of last digits (1 - 9) in the data set conforms to %1$s." ,distribution))
```

Comment:
"The null hypothesis specifies that the first digits (1 - 9) in the data set are distributed according to %1$s."

Benform's law

or 

the uniform distribution